core/geometry/src/curve/internalContexts/EllipticalArcApproximationContext.ts

```diff
@@ -829,8 +832,9 @@ export class EllipticalArcApproximationContext {
         processor.announceArc(arc, undefined, f0, f1);
     };
     const createInnerArc = (f0: number, f1: number, f2: number) => {
+      // This arc starts at f1 and ends at f2
       let fPrev = f0;
-      if (processor.getPreviousFraction)
+      if (processor.getPreviousFraction) // if processor object has a getPreviousFraction method
```

Comment:
Unnecessary. This is an idiom.
```suggestion
      if (processor.getPreviousFraction)
```